resources/views/login-error.blade.php

```diff
@@ -4,7 +4,7 @@
       {{-- <svg class="h-5 w-5 text-red-400" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
         <path fill-rule="evenodd" d="M10 18a8 8 0 100-16 8 8 0 000 16zM8.28 7.22a.75.75 0 00-1.06 1.06L8.94 10l-1.72 1.72a.75.75 0 101.06 1.06L10 11.06l1.72 1.72a.75.75 0 101.06-1.06L11.06 10l1.72-1.72a.75.75 0 00-1.06-1.06L10 8.94 8.28 7.22z" clip-rule="evenodd" />
       </svg> --}}
-	  <x-icon name="far-circle-x" class='w-8 text-red-400' />
+{{--	  <x-icon name="far-circle-x" class='w-8 text-red-400' />--}}
```

Comment:
Not everyone has this icon set.

Reply:
Good catch. 

I was looking to wrap up the beta and get it released alongside Laravel 11 support.

I'll double check and make sure any font awesome icons have been removed.

Reply:
any update?

Reply:
This was fixed in v3.6.1 by #49

Reply:
Can you please remove this from the PR?